tests/units/badge/minter/test_initialization.cairo

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+
+%lang starknet
+
+// Starkware dependencies
+from starkware.cairo.common.cairo_builtins import HashBuiltin, SignatureBuiltin
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.math import assert_not_zero
+
+// Local dependencies
+from src.badge.minter import owner, initializer, getSignerPublicKey, getBadgeContractAddress
+
+@external
+func test_initialization{
```

Comment:
You should do the same as it's done in `tests/units/minter/test_initialization.cairo`